packages/viz/src/lib/style/Classifier.ts

```diff
@@ -0,0 +1,137 @@
+import { CartoStylingError, stylingErrorTypes } from '../errors/styling-error';
+import { NumericFieldStats } from '../types';
+
+export type ClassificationMethod = 'quantiles' | 'stdev' | 'equal';
+
+export class Classifier {
```

Comment:
I thought about having different classes for each classifier, for better testing and isolation. I think we can reuse quite a bit of test specs from VL

Reply:
I think we don't need it right now, so I'll add this extra complexity if this class grows, by the moment I don't see it too large.